utils/src/field_helpers.rs

```diff
@@ -19,19 +30,27 @@ pub trait FieldHelpers<F> {
 
     /// Serialize to bits
     fn to_bits(self) -> Vec<bool>;
+
+    /// Field size in bytes
+    fn size_in_bytes() -> usize
+    where
+        F: PrimeField,
+    {
+        ((F::size_in_bits() as f64) / 8.0).ceil() as usize
```

Comment:
How about avoiding f64 conversion by doing
```
let bits = F::size_in_bits();
match bits % 8 {
    0 => bits / 8,
    _ => bits / 8 + 1,
}
```

Reply:
This makes sense - I'll swap it to something like `bit_count / 8 + (bit_count % 8 != 0) as usize`